packages/ppx_version/ppx_version.0.1/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+name: "snarky"
```

Comment:
I think this should probably be `ppx_version`

Reply:
good catch! I'm wondering why nothing complained : )